test/startservers.py

```diff
@@ -47,14 +47,18 @@ def start(race_detection):
     signal.signal(signal.SIGTERM, lambda _, __: stop())
     signal.signal(signal.SIGINT, lambda _, __: stop())
     global processes
-    for srv in [
+    forwards = [
             [":19091", "publisher.boulder:9091"],
             [":19092", "va.boulder:9092"],
             [":19093", "ca.boulder:9093"],
             [":19094", "ra.boulder:9094"],
             [":19095", "sa.boulder:9095"],
             [":19096", "ca.boulder:9096"],
-    ]:
+            [":19097", "va.boulder:9097"],
+            [":19098", "va.boulder:9098"]
+    ]
+
+    for srv in forwards:
```

Comment:
Actually, thinking about it again: For forwarded ports, it's fine to run listenbuddy to forward to a port that isn't listening. It won't error until someone actually tries to connect to it.